pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -209,9 +212,24 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 		}
 
-		if err := r.reconcileCrashRetention(namespace, cephCluster, cephVersion); err != nil {
-			return reconcile.Result{}, err
+		k8sVersion, err := r.context.Clientset.Discovery().ServerVersion()
```

Comment:
There is a utility function to get the k8s version - `GetK8SVersion()`

Reply:
done.